pkg/ovndbcluster/statefulset.go

```diff
@@ -113,6 +114,32 @@ func StatefulSet(
 			},
 		},
 	}
+	// https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#persistentvolumeclaim-retention
+	statefulset.Spec.PersistentVolumeClaimRetentionPolicy = &appsv1.StatefulSetPersistentVolumeClaimRetentionPolicy{
+		WhenDeleted: appsv1.DeletePersistentVolumeClaimRetentionPolicyType,
+		WhenScaled:  appsv1.RetainPersistentVolumeClaimRetentionPolicyType,
+	}
+	statefulset.Spec.VolumeClaimTemplates = []corev1.PersistentVolumeClaim{
+		corev1.PersistentVolumeClaim{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "etc-ovn",
```

Comment:
Perhaps we should prepend the `instance.Name` to the PVC `Name`?

Reply:
Yes right should be good to add, will check and update.

Reply:
Done